### DISCUSSION TITLE
How to write an alias that uses $HOME env var?

### DISCUSSION THREAD

**Discussion by norman784** [nushell/nushell#5121] [Q&A] [answered]
Hi, I have this alias that I want to migrate to nu

```sh
alias config = "/usr/bin/git --git-dir=$HOME/.dotfiles/ --work-tree=$HOME"
```

I thought this will be an easier one, but it seems that this doesn't work

```nu
alias config = /usr/bin/git --git-dir="$env.HOME/.dotfiles/" --work-tree=$env.HOME
# also tried
alias config = /usr/bin/git --git-dir=( $env.HOME | path join .dotfiles ) --work-tree=$env.HOME
# and
let dotfiles = ( $env.HOME | path join .dotfiles )
alias config = /usr/bin/git --git-dir=$dotfiles --work-tree=$env.HOME
```

Then when executing the script I get `fatal: not a git repository: '($env.HOME | path join '.dotfiles')'` so it seems to me that isn't evaluating what's in the parentheses.

**Comment by kubouch** [comment, marked as answer]
You need to do string interpolation on the parameters: `alias config = /usr/bin/git $'--git-dir=($dotfiles)' $'--work-tree=($env.HOME)'`. Otherwise, it will take "--git-dir=$dotfiles" as a one string without filling in the variable.

**Reply by norman784** [reply]
Thanks for the answer.
